change/@fluentui-react-dialog-73479874-2670-482b-9326-9897d113debb.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33380&commitId=c6249966778d7e90dbeab82de5afd8bb3ac45e32&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|5| Changed|
|Avatar Converged.badgeMask.chromium.png|5| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|983| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|2239| Changed|

</details>


  <details><summary><b>Tree</b> 44 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tree.Actions Dark Mode.chromium.png|1521943| Changed|
|Tree.Default High Contrast.chromium.png|1488223| Changed|
|Tree.Actions High Contrast.chromium.png|1539437| Changed|
|Tree.Default Dark Mode.chromium.png|1467500| Changed|
|Tree.Appearance Dark Mode.chromium.png|1474083| Changed|
|Tree.Actions RTL.chromium.png|1609318| Changed|
|Tree.Appearance High Contrast.chromium.png|1495883| Changed|
|Tree.Appearance RTL.chromium.png|1608985| Changed|
|Tree.Expand Icon Dark Mode.chromium.png|1317128| Changed|
|Tree.Default RTL.chromium.png|1609229| Changed|
|Tree.Default Open Tree Dark Mode.chromium.png|1164243| Changed|
|Tree.Default Open Tree RTL.chromium.png|1608235| Changed|
|Tree.Default Open Tree High Contrast.chromium.png|1134530| Changed|
|Tree.Expand Icon High Contrast.chromium.png|1313115| Changed|
|Tree.Expand Icon RTL.chromium.png|1608748| Changed|
|Tree.Flat Dark Mode.chromium.png|1214524| Changed|
|Tree.Flat High Contrast.chromium.png|1193278| Changed|
|Tree.Flat RTL.chromium.png|1609456| Changed|
|Tree.Flat Tree Multi Selection Dark Mode.chromium.png|1163556| Changed|
|Tree.Flat Tree Multi Selection RTL.chromium.png|1609190| Changed|
|Tree.Flat Tree Multi Selection High Contrast.chromium.png|1133865| Changed|
|Tree.Layout Dark Mode.chromium.png|1522056| Changed|
|Tree.Flat Tree Single Selection Dark Mode.chromium.png|1163489| Changed|
|Tree.Flat Tree Single Selection RTL.chromium.png|1609190| Changed|
|Tree.Flat Tree Single Selection High Contrast.chromium.png|1133719| Changed|
|Tree.Persona High Contrast.chromium.png|1013742| Changed|
|Tree.Layout RTL.chromium.png|1609095| Changed|
|Tree.Persona Dark Mode.chromium.png|1029512| Changed|
|Tree.Size Dark Mode.chromium.png|1266378| Changed|
|Tree.Layout High Contrast.chromium.png|1539629| Changed|
|Tree.Persona RTL.chromium.png|1606353| Changed|
|Tree.Size High Contrast.chromium.png|1253821| Changed|
|Tree.actions.chromium.png|1609386| Changed|
|Tree.Size RTL.chromium.png|1608382| Changed|
|Tree.default.chromium.png|1609312| Changed|
|Tree.default open tree.chromium.png|1608365| Changed|
|Tree.expand icon.chromium.png|1608843| Changed|
|Tree.appearance.chromium.png|1609041| Changed|
|Tree.flat tree multi selection.chromium.png|1609161| Changed|
|Tree.layout.chromium.png|1609167| Changed|
|Tree.flat.chromium.png|1609468| Changed|
|Tree.flat tree single selection.chromium.png|1609239| Changed|
|Tree.persona.chromium.png|1606412| Changed|
|Tree.size.chromium.png|1608472| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>